packages/hub/cli.ts

```diff
@@ -33,20 +34,19 @@ yargs(hideBin(process.argv))
       console.error(printCompilerError(err));
 
       Sentry.captureException(err, { tags: { alert: 'web-team' } });
-      // Make sure that we send all exceptions before exiting
-      Sentry.getCurrentHub()!
-        .getClient()!
-        .close(5000)
-        .then((result) => {
-          if (!result) {
-            console.error('Failed to send error to Sentry because we hit the timeout.');
-          }
-
-          // eslint-disable-next-line no-process-exit
-          process.exit(1);
-        });
-    } else {
-      // eslint-disable-next-line no-process-exit
-      process.exit(1);
+      let sentryClient = Sentry.getCurrentHub()?.getClient();
+
+      if (sentryClient) {
+        // Make sure that we send all exceptions before exiting
+        let result = await sentryClient.close(5000);
```

Comment:
We're waiting 5 seconds for Sentry to send its exceptions, which feels like it should be enough for most CLI commands